cegui/src/text/LegacyTextParser.cpp

```diff
@@ -257,13 +276,25 @@ void LegacyTextParser::processControlString(const std::u32string& ctrlStr, std::
 
     if (valueValid && key == ImageTagName)
     {
+        const String val = ctrlStr.substr(valueStart, valueEnd - valueStart);
+        if (!ImageManager::getSingleton().isDefined( val )) {
```

Comment:
Here and below - please follow the code style. We don't use spaces inside round brackets.

Reply:
fixed